create_infotexts.py

```diff
@@ -100,6 +100,22 @@ def load_places_mapping():
 
     return places_dict
 
+def create_commons_filename(metadata, fotonr):
+    """Creates commons filename according to https://phabricator.wikimedia.org/T156612 and ouputs to new_dict
+    :type populated_dict: dictionary
+    """
+
+    if not metadata[fotonr]["Beskrivning"] == "":
+        cleaned_fname = helpers.format_filename(metadata[fotonr]["Beskrivning"], "SMVK-MM-Cypern", metadata[fotonr]["Fotonummer"])
+        #print("Fname using BatchUploadTools: {}".format(cleaned_fname))
+    else:
+        # TODO: fix alternative description according to https://phabricator.wikimedia.org/T156612#3008806 [Issue: https://github.com/mattiasostmar/SMVK-Cypern_2017-01/issues/9]
+        beskr = "Svenska Cypernexpeditionen 1927-1931"
```

Comment:
Such a default description could be suitable if we only have a handful of images without a `metadata[fotonr]["Beskrivning"]`. Any more and all the filenames will start to look the same.

It's ok to keep for now if you create a Task to generate more rich descriptions and link it from here (i.e. instead of the closed issue in the todo above )